litellm/litellm_core_utils/litellm_logging.py

```diff
@@ -1090,6 +1091,27 @@ def success_handler(
                             end_time=end_time,
                             print_verbose=print_verbose,
                         )
+                    if callback == "langtrace":
```

Comment:
hey @alizenhom i don't think we need an explicit callback, since you're implementation is CustomLogger compatible 

this should be handled here - https://github.com/BerriAI/litellm/blob/3007f0344d3a84c019377a27935bcc786bdaf253/litellm/litellm_core_utils/litellm_logging.py#L1947

Reply:
agreed, will remove it from callbacks and handle it as mentioned